docker/sandbox/flyte-entrypoint-dind.sh

```diff
@@ -19,6 +19,12 @@ monitor() {
 
 # Start docker daemon
 echo "Starting Docker daemon..."
+# Remove docker PID if exist, Used for restart
+if [ -f "/var/run/docker.pid" ]
+then
+    rm /var/log/dockerd.log
+fi
```

Comment:
Why this is only needed in dind and not the default.

Reply:
reverted